git/objects/commit.py

```diff
@@ -335,8 +336,70 @@ def stats(self) -> Stats:
         return Stats._list_from_string(self.repo, text)
 
     @property
-    def trailers(self) -> Dict:
-        """Get the trailers of the message as dictionary
+    def trailers(self) -> Dict[str, str]:
+        """Get the trailers of the message as a dictionary
+
+        Git messages can contain trailer information that are similar to RFC 822
+        e-mail headers (see: https://git-scm.com/docs/git-interpret-trailers).
+
+        WARNING: This function only returns the latest instance of each trailer key
+        and will be deprecated soon. Please see either ``Commit.trailers_list`` or ``Commit.trailers_dict``.
+
+        :return:
+            Dictionary containing whitespace stripped trailer information.
+            Only the latest instance of each trailer key.
+        """
+        return {
+            k: v[0] for k, v in self.trailers_dict.items()
+        }
+
+    @property
+    def trailers_list(self) -> List[str]:
```

Comment:
To me it seems this method would be more useful if the keys and values would be returned as a tuple, i.e. `List[(str, str)]`. Is there a good reason not to do that?

PS: *It's probably OK to always trim whitespace as it should never be significant.*

Reply:
Agreed - done [abde3ea](https://github.com/gitpython-developers/GitPython/pull/1576/commits/abde3eafd293e8fa2ef2dc22d58ba5d80f1702e9)